public/css/mixin/progress-bar.less

```diff
@@ -15,17 +15,22 @@
       padding: 1.25em 10%;
     }
 
+    .positioned {
+      position: absolute;
+    }
+
     .bubble {
       .rounded-corners(.25em);
       background-color: @body-bg-color;
       border: 1px solid @gray-light;
-      position: absolute;
-      bottom: 0;
+      position: relative;
       box-shadow: 0 0 1em 0 rgba(0, 0, 0, .1);
-      margin-left: -4em;
       padding: .25em .5em;
       text-align: center;
-      width: 8em;
```

Comment:
If you find a way to make apply `width: auto` to the bubbles and still make the alignment of the carets (`.bubble:before`) work, please also add a `max-width`.

Please keep in mind, that the content width can vary, when being translated.

![Screen Shot 2021-09-28 at 13 27 21](https://user-images.githubusercontent.com/6977434/135078276-3d9deae6-bb6e-403a-873e-8f0e64f6c94b.png)

It's also important to prevent `.start` and `.end` `.bubble` being obscured completely, since the user needs to be able to hover the obscured bubble to see its content.

Reply:
> It's also important to prevent `.start` and `.end` `.bubble` being obscured completely

The bubble can also be displayed by moving the mouse pointer over the bubble caret. 🤔 



> please also add a `max-width`

Why is this necessary? Bubble only contains the timestamp, and it can't be that long anyway.

Reply:
> > It's also important to prevent `.start` and `.end` `.bubble` being obscured completely
> 
> The bubble can also be displayed by moving the mouse pointer over the bubble caret. 🤔

I think, it's pretty obvious on the screenshot, that when one bubble is completely obscured, it's almost impossible to spot. It's also not quite clear at first sight, which caret belongs to which bubble.

And besides spotting it in the first place, the caret alone is a much too small target and it will be pretty annoying for the user to navigate the cursor there.

> > please also add a `max-width`
> 
> Why is this necessary? Bubble only contains the timestamp, and it can't be that long anyway.

If you can ensure, that it's wide enough for all date formats in all languages, I'm fine with having no `max-width`.